README.md

```diff
@@ -76,46 +72,7 @@ To Register an app:
 5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 2B below.
 6. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app_bundle_id>://auth`. Replace <app_bundle_id> with the **Bundle Identifier** for your application. 
 7. Hit the **Save** button in the top left, to save these updates.
-
-## 2B: Configure your application
-
-1. Update your application's redirect URI scheme in the  `Info.plist` file by replacing `msauth.com.microsoft.identitysample.MSALiOS` . Redirect URI scheme follows the format `msauth.[app_bundle_id]`. Make sure to substitue [app_bundle_id] with the **Bundle Identifier** for your application. 
-
-```xml
-<key>CFBundleURLTypes</key>
-<array>
-  <dict>
-    <key>CFBundleURLSchemes</key>
-    <array>
-      <string>msauth.[app_bundle_id]</string>
-    </array>
-  </dict>
-</array>
-```
-
-2. Configure your application defaults
-
-In the `ViewControler.swift` file, update the `kClientID` variable with your Application (client) ID.
-
-```swift
-    // Update the client ID below with the one you received in the portal. Below ID is for running the sample app only.
-    
-    let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
-```
-
-In the `ViewControler.swift` file, update the `kAuthority` variable with your Azure AD and Microsoft Graph endpoints for your national cloud. For global access, use following values:
-
-```objective-c
-let kGraphEndpoint = "https://graph.microsoft.com/"
-let kAuthority = "https://login.microsoftonline.com/common"
-```
-
-Other endpoints are documented [here](https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints). For example, to run the sample with AzureAD Germany, use following:
-
-```objective-c
-let kGraphEndpoint = "https://graph.microsoft.de/"
-let kAuthority = "https://login.microsoftonline.de/common"
-```
+8. Click **Make these changes for me** and then download the code sample for iOS 
```

Comment:
Should we still keep manual configuration options as a reference? (in case someone is trying to configure it for an existing project)

Reply:
Make sense, those are still valid if devs starts with their own project, will revert and make a meaningful title

Reply:
Great, thank you!

Reply:
Add a separated section for own projects